### ISSUE TITLE
I get this Response Error when making a request: Error: Invalid character in header content ["pm-o0"]

### ISSUE THREAD

**Issue by michaelwiles** [a85/POSTMan-Chrome-Extension#12435]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

I make a call to and endpoint with postman and I immediately get this response:

Error: Invalid character in header content ["pm-o0"]

This is the console log:
POST https://`<REDACTED>`?appId=`<REDACTED`>&language=en
Error: Invalid character in header content ["pm-o0"]
Network
Request Headers
Content-Type: application/json
User-Agent: PostmanRuntime/7.33.0
Accept: */*
Cache-Control: no-cache
Postman-Token: 421addb0-d572-4222-af86-74cd1fd13ebf
Host: `<REDACTED>`
Accept-Encoding: gzip, deflate, br
Connection: keep-alive
Cookie: `<REDACTED>`
Request Body
{
"appSecret": "`<REDACTED>`",
"email": "`<REDACTED>`",
"password": "`<REDACTED>`"
}

I take the call and run it with curl (as postman generates it) it works fine.

Based on my searching it seems to have something to do with CORS but I have no idea what... I can't see that rogue header it is talking about in the response when I use curl.

### Steps To Reproduce

I'm not sure I can put anything useful here as I'm assuming on my specific endpoint surfaces this issue and I can't provide credentials to call it successfully.

### Screenshots or Videos

![image](https://github.com/postmanlabs/postman-app-support/assets/1850356/61e7762e-1fda-4a1a-ac95-9e43ccbf20f4)

### Operating System

Linux

### Postman Version

10.19.9-231025-0520

### Postman Platform

Postman Web

### User Account Type

Signed In User

**Comment by codenirvana** [comment]
@michaelwiles Are their cookies stored for this hostname?
If yes, can you copy the cookie values for reference and, delete all the cookies and check if that fixes this issue?

**Comment by michaelwiles** [comment]
Where do I look for cookies when you say are there cookies stored this host name... In the browser? Why would those be applied by the postman call? Does postman have cookies?

**Comment by codenirvana** [comment]
Yes, Postman stores cookies associated with a host, which are accessible via:
![Screenshot 2023-10-26 at 6 45 44 PM](https://github.com/postmanlabs/postman-app-support/assets/4865836/3cddff10-9237-48ab-b305-2c14759e2788)

If you see the hostname listed in the cookie modal, try clearing all the cookies and retry.
![Screenshot 2023-10-26 at 6 46 23 PM](https://github.com/postmanlabs/postman-app-support/assets/4865836/73ea77a5-fa91-4f32-b2cb-002c46c54b94)

Could you make the cookie content before deleting it to help debug the issue later?

**Comment by michaelwiles** [comment]
There are no cookies....

So the plot thickens.

There's a few scenarios to outline:
1. Switching to the Desktop Agent from the cloud agent everything works
2. Using the cloud agent with a call that is successful also works
3. But if I send an invalid payload I get the error though I've no idea what's causing it.

I _think_ the server is sending something strange back but the cloud agent is throwing an error when this happens. I cannot see that the request is not getting to the destination server. However, when I look at the response headers when running with the desktop agent I don't see anything amiss (I don't see that weird header it is talking about).

**Comment by Tasya201** [comment]
yes

**Comment by saket-ranjan-010** [comment]
@michaelwiles Are you using any specific cipher suites with your request by any chance? If yes, can you try resetting the option to default and then try to send the request again.

You can find this setting under Request tab -> Settings -> Cipher suite selection
<img width="1067" alt="Screenshot 2023-12-22 at 12 03 39 PM" src="https://github.com/postmanlabs/postman-app-support/assets/116867391/a31c5cee-ee2c-4dff-8dd0-627e9ac6cc60">

**Comment by geertschuring** [comment]
I'm running into the same problem. Can't get my finger on why it sometimes works but fails 90% of the time. This makes the Postman cloud unusable for me...

**Comment by saket-ranjan-010** [comment]
Hey @geertschuring , if possible can you send me the HAR logs from the session when your request fails with this error? It'll help me in figuring out what could be the reason for this issue. You can share the logs in this thread or send it over to this email: help@postman.com. Thanks!
